src/main/com/fulcrologic/fulcro/algorithms/normalized_state_helpers.cljc

```diff
@@ -24,17 +37,82 @@
   => [:person/id 3 :person/first-name]
   ```
   "
-  [state path]
-  [map? vector? => vector?]
-  (loop [[h & t] path
-         new-path []]
-    (if h
-      (let [np (conj new-path h)
-            c  (clojure.core/get-in state np)]
-        (if (eql/ident? c)
-          (recur t c)
-          (recur t (conj new-path h))))
-      new-path)))
+  ([state path]
+   [map? vector? => vector?]
+   (loop [[h & t] path
+          new-path []]
+     (if h
+       (let [np (conj new-path h)
+             c (clojure.core/get-in state np)]
+         (if (eql/ident? c)
+           (recur t c)
+           (recur t (conj new-path h))))
+       ;;new-path )))
+       (if (not= path new-path)
+         new-path
+         nil))))
+
+  ([state path not-found]
```

Comment:
You should call this function itself instead of coding it twice.

Reply:
Yup, this is resolved.